presto-main/src/test/java/io/prestosql/sql/query/QueryAssertions.java

```diff
@@ -368,7 +368,7 @@ public String toStringOf(Object object)
             {
                 if (object instanceof SqlTimestamp) {
                     SqlTimestamp timestamp = (SqlTimestamp) object;
-                    return String.format(
+                    return String./*suppress code style*/format(
```

Comment:
Here comes the cost of this. There is a `format` method defined in `StandardRepresentation`

Reply:
😔

Reply:
maybe it's worth/possible to configure some hints (airlift or just IDE) instead of hard restriction in https://github.com/airlift/airbase/pull/247 ?

Reply:
That’s why I don’t like the blanket ban on String.format. There are places where you just want to use that name and delegate to String.format for convenience, and this prevents it.

I don’t see anything wrong with having qualified calls to String.format.